linter.py

```diff
@@ -45,6 +45,21 @@ class ESLint(NodeLinter):
         'html': 'source.js.embedded.html'
     }
 
+    def __init__(self, view, syntax):
+        """Initialize a new NodeLinter instance."""
+        super(ESLint, self).__init__(view, syntax)
+
+        # Now we have the package.json, if any
+        if self.manifest_path:
+            pkg = self.get_manifest()
+            # If eslint-html-plugin is specified, we assume it is in use and
+            # disable our own html selectors.
+            if (('dependencies' in pkg and
+                 'eslint-plugin-html' in pkg['dependencies']) or
+                ('dependencies' in pkg and
+                 'eslint-plugin-html' in pkg['devDependencies'])):
+                self.selectors.pop('html')
```

Comment:
Here you mutate the `cls.selectors` mapping bc self here inherits the cls members. Since `selectors` only holds 'html' at this point, it would be good enough to just assign a new dict `self.selectors = {}`. Otherwise you would get that dict, copy it and then mutate and assign. `selectors = self.selectors.copy(); selectors.pop(...); self.selectors = selectors`. In a way, similar to js, if you don't want to mutate a member on the prototype.